app/js/lib/server/index.js

```diff
@@ -21,7 +21,7 @@ const p = Object.freeze({
 
 export default class Server extends EventDispatcher {
   constructor({ settings, net } = {}) {
-    super(['online', 'push-message']);
+    super(['logged-in', 'online', 'push-message']);
```

Comment:
it's still ambiguous :) for me the "login" event means "when I start the login request". Not when it ended successfully. Maybe that's just me ?